soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -66,6 +66,20 @@ config SOC_IT8XXX2_CPU_IDLE_GATING
 	  gated by individual drivers. When this option is disabled, CPU idle
 	  mode is always permitted.
 
+config SOC_IT8XXX2_JTAG_ENABLE
+	bool "The JTAG module clock gating is configured here"
+	help
+	  This option determines if the JTAG module would be enabled or not.
+
+config SOC_IT8XXX2_ENABLE_FPU
+	bool "FPU Enable"
+	default y if FPU
+	help
+	  Enable the on-chip FPU. If disabled, the FPU will be clock gated and floating-point
+	  instructions will (do something, replace with what actually happens).
```

Comment:
Please update this to describe what actually happens if the FPU is used while disabled.

Reply:
Hi Peter,

Apologetic for my carelessness, I've updated it as files changed.

Reply:
@tari 

If there's any further suggestion, please kindly let me know.
Thanks again.